src/styles/_helpers.scss

```diff
@@ -1,3 +1,28 @@
 .rochester {
-  font-family: $rochester;
-}
\ No newline at end of file
+	font-family: $rochester;
+}
+
+#rochester-p {
```

Comment:
please unly use ids for top level layouts

Reply:
Noted sir.